taskcluster/translations_taskgraph/parameters.py

```diff
@@ -43,9 +43,9 @@ def get_defaults(_):
                     "dim-vocabs": "1000 1000",
                 },
                 "training-teacher": {
-                    "disp-freq": "1",
-                    "save-freq": "25",
-                    "valid-freq": "50",
+                    "disp-freq": "3",
```

Comment:
suggestion: We probably don't want to take this change (I think you needed it for testing though).

(I'll defer to @eu9ene on whether or not this is blocking.)